src/components/Modal/Comment/Comment.tsx

```diff
@@ -95,22 +84,24 @@ function Comment({ commentData, onDelete, onUpdate }: CommentProps) {
                   취소
                 </button>
               </>
-            ) : (
+            ) : commentData?.author.id === user.id ? (
               <button
                 type="button"
                 className={styles.btn}
                 onClick={handleEditToggle}
               >
                 수정
               </button>
-            )}
-            <button
-              type="button"
-              className={styles.btn}
-              onClick={() => onDelete(commentData.id)}
-            >
-              삭제
-            </button>
+            ) : null}
+            {commentData?.author.id === user.id ? (
+              <button
+                type="button"
+                className={styles.btn}
+                onClick={() => onDelete(commentData ? commentData.id : 0)}
```

Comment:
```suggestion
                onClick={() => onDelete(commentData ? commentData.id : 0)}
```
0을 삭제하는 경우는 어떤 경우인가요??